document-store/src/main/java/org/hypertrace/core/documentstore/mongo/update/parser/MongoPullAllOperationParser.java

```diff
@@ -0,0 +1,24 @@
+package org.hypertrace.core.documentstore.mongo.update.parser;
+
+import static org.hypertrace.core.documentstore.model.subdoc.UpdateOperator.REMOVE_ALL_FROM_LIST;
+import static org.hypertrace.core.documentstore.mongo.MongoUtils.merge;
+
+import com.mongodb.BasicDBObject;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import org.hypertrace.core.documentstore.model.subdoc.UpdateOperator;
+
+@AllArgsConstructor
+public class MongoPullAllOperationParser extends MongoOperationParser {
```

Comment:
nit/thought: `Pull` is internal to Mongo. So, if we can keep the name as our Update Operator, we may able to map them easily while reading.
 e.g naming as `MongoRemoveAllFromListOperationParser`

Reply:
Sure.